bundles/BasicEditor.ts

```diff
@@ -5,12 +5,27 @@ import { LineBreak } from '../packages/plugin-linebreak/LineBreak';
 import { Heading } from '../packages/plugin-heading/Heading';
 import { Paragraph } from '../packages/plugin-paragraph/Paragraph';
 import { List } from '../packages/plugin-list/List';
+import { Span } from '../packages/plugin-span/Span';
+import { Bold } from '../packages/plugin-bold/Bold';
+import { Italic } from '../packages/plugin-italic/Italic';
+import { Underline } from '../packages/plugin-underline/Underline';
 
 export class BasicEditor extends JWEditor {
     constructor(editable?: HTMLElement) {
         super(editable);
         this.loadConfig({
-            plugins: [Dom, Char, LineBreak, Heading, Paragraph, List],
+            plugins: [
+                Dom,
+                Char,
+                LineBreak,
+                Heading,
+                Paragraph,
+                List,
+                Span,
+                Bold,
+                Italic,
+                Underline,
```

Comment:
It's a bit weird that this list does not include `Format` as a Plugin since it's the one who is suppose to declare the `InlineNode` that `CharNode` is supposed to extend, but maybe that is introduced in a later commit ?